scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -445,19 +445,24 @@ def build_test_packages(change_ids, obs_linked_project, home_project,
             try_branches.insert(0, c.target)
         processed_changes.append(c.id)
 
+        # skip packages that don't have asssociated RPMs
```

Comment:
(nit) This could go immediately after getting the GerritChange (line 441), which will avoid potentially erroneous entries in try_branches.